tools/chpldoc/findUndocumentedSymbols.py

```diff
@@ -130,6 +130,15 @@ def is_nodoc(node: dyno.AstNode) -> bool:
     ):
         return True
 
+    # MultiDecl apparently supports comments inside of it
+    # so it requires the extra `hasattr` check
+    if (
+        isinstance(node, dyno.MultiDecl)
+        and any(n.name().startswith("chpl_") for n in node if hasattr(n, "name"))
+        and "chpldoc ignore chpl prefix" not in node.pragmas()
+    ):
+        return True
```

Comment:
Won't this skip multi-decls like the following:

```Chapel
var chpl_x = a,
    x = b;
```

Perhaps `chpl_`-based filtering needs to happen at the point where we extract names, not at the node level.

Reply:
I think this is a bigger design question for chpldoc, especially since chpldoc currently won't generate docstrings for MultiDecls. And the check for `symbolNameBeginsWithChpl` in chpldoc always returns false for MultiDecls anyways.

We don't have the situation you described in our modules, and I think I would rather keep the code cleaner as-is with an extra comment about this rather than making it a special case in the name extraction

Reply:
A comment is fine by me.